src/index.html

```diff
@@ -11,7 +11,157 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="infoblock" name="maininfo">
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input type="text" name="surname">
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input
+          type="text"
+          name="name"
+          autocomplete="off"
+          >
```

Comment:
[point 3](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)

```suggestion
          <input
            type="text"
            name="name"
            autocomplete="off"
          >
```